### PR TITLE
Added TheMediaGridNM Bid Adapter

### PR DESCRIPTION
**Pull request by TheMediaGrid**
## Type of change

- [x] New bidder adapter  

## Description of change

Added new TheMediaGridNM Bid Adapter

- test parameters for validating bids
```
{
   bidder: "gridNM",
   params: {
       source: 'jwp',
       secid: '11',
       pubid: '22',
       video: {
           mimes: ['video/mp4', 'video/x-ms-wmv'],
           protocols: [1,2,3,4,5,6]
       }
   }
}
```

- grid-tech@themediagrid.com
- [x] official adapter submission

- A link to a PR on the docs repo - https://github.com/prebid/prebid.github.io/pull/1865

### CODE REVIEW

**Comment by TheMediaGrid** [comment]
Hi @aleksatr!
Could you please review this?
Thanks in advance!

File: modules/gridNMBidAdapter.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+Grid bid adapter supports Banner and Video (instream and outstream).
+
+# Test Parameters
+```
+    var adUnits = [
+           {
+               code: 'test-div',
+               mediaTypes: { 
+                   video: {
+                       playerSize: [728, 90]
```
Comment (aleksatr) [inline modules/gridNMBidAdapter.md]: maybe you want to add `context: "outstream"` here, to get the running example?

File: modules/gridNMBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+  noBid: 'Array of bid objects is empty',
+  noPlacementCode: 'Can\'t find in requested bids the bid with auid - ',
+  emptyUids: 'Uids should be not empty',
+  emptySeatbid: 'Seatbid array from response has empty item',
+  emptyResponse: 'Response is empty',
+  hasEmptySeatbidArray: 'Response has empty seatbid array',
+  hasNoArrayOfBids: 'Seatbid from response has no array of bid objects - '
+};
+
+// const KEYS_FOR_PARAMS = ['secid','pubid','pubdata','floorcpm','source','video'];
```
Comment (aleksatr) [inline modules/gridNMBidAdapter.js]: Can this be removed (L27-L28)?

File: modules/gridNMBidAdapter.js [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+          payload.gdpr_applies =
+            (typeof bidderRequest.gdprConsent.gdprApplies === 'boolean')
+              ? Number(bidderRequest.gdprConsent.gdprApplies) : 1;
+        }
+        if (bidderRequest.uspConsent) {
+          payload.us_privacy = bidderRequest.uspConsent;
+        }
+      }
+
+      /* const content = {};
```
Comment (aleksatr) [inline modules/gridNMBidAdapter.js]: Can this be removed (L97-L113)?

**Comment by aleksatr** [comment]
> Hi @aleksatr!
> Could you please review this?
> Thanks in advance!

Hey @TheMediaGrid , sorry for the delay, if you can check comments that I left, everything else looks good to me.

**Pull request merged**